microcosm/tests/test_object_graph.py

```diff
@@ -163,7 +162,20 @@ def test_get_component_name_function():
     graph.use("parent")
     graph.lock()
 
+    assert_that(get_component_name(graph, graph.parent), is_(equal_to("parent")))
+
+
+def test_describe_object_graph():
+    graph = create_object_graph(
+        name="test", description="A description of the object graph."
+    )
     assert_that(
-        get_component_name(graph, graph.parent),
-        is_(equal_to("parent"))
+        graph.metadata.description, is_(equal_to("A description of the object graph."))
```

Comment:
Sounds good. Do you have a specific case in mind to use this in for context? 🤔

Reply:
The description will be set on the `sphinx` graph object and used to generate a top-level description of its API in the auto-generated swagger docs.